graf2d/asimage/inc/TASPngWriter.h

```diff
@@ -0,0 +1,35 @@
+#ifndef ROOT_TASPngWriter
+#define ROOT_TASPngWriter
+
+#include <string_view>
+#include <vector>
+
+class TASPngWriter
```

Comment:
Would be nice to have minimal docu about this class.
Maybe just few comment lines and link on the source

Reply:
Done.